jquery.confirm.js

```diff
@@ -69,7 +69,10 @@
         // Default options
         var settings = $.extend({}, $.confirm.options, {
             confirm: function () {
-                if (dataOptions.submitForm){
+                if (dataOptions.submitForm
+                    || (typeof dataOptions.submitForm == "undefined" && options.submitForm)
+                    || (typeof dataOptions.submitForm == "undefined" && typeof options.submitForm == "undefined" && $.confirm.options.submitForm)
+                ) {
```

Comment:
You want to make sure that dataOptions override options and both of them override $.confirm.options

(short answer, i'm writing an example and will edit to make my self clear)